pyradiosky/skymodel.py

```diff
@@ -2391,6 +2501,24 @@ def read_fhd_catalog(
 
         return
 
+    @classmethod
+    def from_idl_catalog(cls, filename_sav, **kwargs):
```

Comment:
I don't think we need this one. The method it's calling is deprecated in favor of `read_fhd_catalog` because that name is more appropriate -- it was always for FHD catalogs, not for some generic IDL catalog which isn't a thing. I think it was called IDL because FHD is written in IDL, but that makes it sound much more generic than it is.